lib/CompilerInterfaceTypes.h

```diff
@@ -14,6 +14,19 @@
 #include <cstring>
 #include <stdint.h>
 
+namespace proteus {
+
+enum RuntimeConstantTypes : int32_t {
```

Comment:
why inherit from, int32_t, just to enforce size?

Reply:
Yes, to match the expected data type from the pass